js/post/update_counts.js

```diff
@@ -1,8 +1,14 @@
+creditCountSpec = 0;
+creditCountMaj = 0;
+creditCountMin = 0;
+creditCount300and400 = {'Spec': 0, 'Maj': 0, 'Min': 0};
+
+
 /**
  * Updates number of completed courses in Specialist.
 **/
 function updateCompletedSpecCourses () {
-    'use-strict';
+    'use-strict'
```

Comment:
Why did you remove the semi colons? From my note last week, this should also be `use strict;`.